examples/gno.land/p/demo/simpledao/propstore.gno

```diff
@@ -59,19 +68,26 @@ func (p *proposal) IsExpired() bool {
 	return false // this proposal never expires
 }
 
-func (p *proposal) Render() string {
+func (p *proposal) Render(idx int, authorUsername string) string {
 	// Fetch the voting stats
 	stats := p.Stats()
 
-	output := ""
-	output += ufmt.Sprintf("Author: %s", p.Author().String())
-	output += "\n\n"
-	output += p.Description()
-	output += "\n\n"
-	output += ufmt.Sprintf("Status: %s", p.Status().String())
-	output += "\n\n"
-	output += ufmt.Sprintf(
-		"Voting stats: YES %d (%d%%), NO %d (%d%%), ABSTAIN %d (%d%%), MISSING VOTE %d (%d%%)",
+	var out bytes.Buffer
+
+	out.WriteString(ufmt.Sprintf("# Proposal #%d - %s\n\n", idx, p.title))
+
+	out.WriteString("## Description\n\n")
+	if strings.TrimSpace(p.description) != "" {
+		out.WriteString(ufmt.Sprintf("%s\n\n", p.description))
+	} else {
+		out.WriteString("No description provided.\n\n")
+	}
+
+	out.WriteString("## Proposal information\n\n")
+	out.WriteString(ufmt.Sprintf("#### Status: %s\n\n", strings.ToUpper(p.Status().String())))
+
+	out.WriteString(ufmt.Sprintf(
+		"#### Voting stats:\n- YES %d (%d%%)\n- NO %d (%d%%)\n- ABSTAIN %d (%d%%)\n- MISSING VOTE %d (%d%%)\n",
```

Comment:
`MISSING VOTES`?

Reply:
Used to be like that, I assumed there was a reason; changed it: [1a9c7ed](https://github.com/gnolang/gno/pull/3096/commits/1a9c7ed19b2e76fa31488538c2eb15841fbbcc68)